src/Engine/Component/Script.cpp

```diff
@@ -45,35 +45,41 @@ Json::Value Script::Save() const {
     return component;
 }
 
-void Script::FillPropertyMap() {
+void Script::AddToPropertyMap(std::string &name, int type, int size, void* data) {
 
-    int propertyCount = instance->GetPropertyCount();
+    if(!IsInPropertyMap(name, type))
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') keyword 'if' not followed by a single space [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=vera%2B%2B%3AT008)